src/pages/FarmPage/FarmsList.tsx

```diff
@@ -487,37 +457,37 @@ const FarmsList: React.FC<FarmsListProps> = ({ bulkPairs, farmIndex }) => {
           ))}
         </Box>
       )}
+      {(farmIndex === GlobalConst.farmIndex.LPFARM_INDEX && !stakingInfos) ||
```

Comment:
We shouldn't care about the index when checking if the data isn't loaded. 
If the data isn't loaded.

if(!stakingInfos || !dualStakingInfos)

Reply:
If I remove the index check it will cause the loading condition true event if one of the list is loaded

Reply:
Hmm you're right you can ignore that comment.
 I misinterpreted.  I was under the impression that only one would be loaded based on the index.